### ISSUE TITLE
Need help find mistake in my custom config.

### ISSUE THREAD

**Issue by alxcool** [crazysnailboy/VillagerTrades#20]
Hi! I really enjoy using your mod in singleplayer 1.12.2 (old launcher), forge 2668.

I may need some help to fix my custom config as the game / server properly load, but without my extra configs.. Here the 3 files I use :.

# Configuration file outside folder

general {
    #  [default: [minecraft:emerald]]
    S:currencyItems <
        minecraft:emerald
     >

    #  [default: true]
    B:loadTradesFromJar=true

    #  [default: true]
    B:loadVillagersFromJar=true

    #  [default: true]
    B:sortTrades=true
}

# In trade section
{ 
	Profession: "vtt:tinkerer", 
	Career:"tinkerer",

	Offers:{
		Recipes:[
		
			{action:"add", buy:{id:"minecraft:bone",Count:{min:12,max:20}},sell:{id:"minecraft:emerald",Count:1}, CareerLevel:1},
			{action:"add", buy:{id:"minecraft:emerald",Count:{min:1,max:2}},sell:{id:"minecraft:stone",Count:8}, CareerLevel:1}
			
		]
	}

}

# In profession section
{
	profession: {
		name: "vtt:tinkerer",
		texture: "vtt:textures/entity/villager/tinkerer.png",
		zombieTexture: "vtt:textures/entity/zombie_villager/zombie_tinkerer.png"
	},
	careers: [
		"tinkerer"
	]
}

Note : I had to create the tinkerer's trade table from scratch as it was not included by default. A villager wearing blue cloth have recently join one of my villages, and I was interested to include him a few trades.

**Comment by alxcool** [comment]
Oh forgot to mention I used both
VillagerTrades-1.12-0.6.4
and
borntobe-1.12.2-1.0

**Comment by Echoweaver** [comment]
Hey, I'm not the dev, but I've done a lot of trade tables.

Did you create the trade table for the villagers AFTER the villager generated in your game?

Once a villager spawns, its trade table is fixed. So see changes, you need to generate a new villager.

**Comment by Flytre** [comment]
Hey, the Json seems to be valid. How did you get borntobe to work, it crashes the game whenever I open a world with it.

**Comment by alxcool** [comment]
Hi! First sorry it took me years to read these comments, I'l close the topic now, and thk! And yes unfortunately I've created the config files, after having a villager to spawn. And thk that will be quite useful for me to learn that trades tables are fixed once villager have spawned ^^, that explain a lots about those custom trades offers by lucky blocks's events.
Fun fact, if you feed your lucky villagers (drop potatoes at his feet), this one will not despawn and will stay in his chunk for a very long time!

About borntobe... this mod have a dependency.
As a minimum to avoid crash, I use :
VIL+borntobe-1.12.2-1.0
VillagerTrades-1.12-0.6.4
Note I've renamed borntobe to make sure not to forget to use it with villagertrades as dependency!
